tests/integration_tests/test_subtensor.py

```diff
@@ -75,19 +75,15 @@ def test_network_overrides( self ):
         assert subtensor.endpoint_for_network() == bittensor.__nobunaga_entrypoint__
 
     def test_connect_no_failure( self ):
-        self.subtensor.connect(timeout = 1, failure=False)
-
+        self.subtensor.substrate
+        assert True
 
     def test_connect_success( self ):
-        success = self.subtensor.connect()
-        assert success == True
-
-
-    def test_connect_fail( self ):
-        self.subtensor.substrate=None
-        with pytest.raises(RuntimeError):
-            self.subtensor.connect()
-
+        try:
+            self.subtensor.substrate
+            assert True
+        except:
+            assert False
```

Comment:
Can this test pass with the previous test passing?

Sounds a little bit confusing for me. 
What do you think about adding assertions with more semantic to see what  `self.subtensor.substrate` is about?

Reply:
I'm pretty certain you don't need the assert
It should fail on error that isn't caught